### PR TITLE
feat: update fonts; add gnome-terminal, git-gui, gitk and their desktop icons

### LINKED ISSUE #144: VNC terminal the powerline prompt is missing some fonts
In the VNC image, the powerline prompt is missing some fonts:

image

should be:

image

_Originally posted by @rokroskar in https://github.com/SwissDataScienceCenter/renkulab-docker/issues/143#issuecomment-814121502_

### PR DESCRIPTION
**Pull request by gavin-k-lee**
closes #144

### CODE REVIEW

File: docker/vnc/Dockerfile [hunk 1]
```diff
@@ -17,7 +17,10 @@ RUN apt-get update \
         xfce4-settings \
         xorg \
         xubuntu-icon-theme \
-        xterm \
+        gnome-terminal \
+        ttf-dejavu \
+        git-gui \
```
Comment (rokroskar) [inline docker/vnc/Dockerfile:22]: we may want to add `gitk` here otherwise the visualization of the git history doesn't work

Comment (gavin-k-lee) [reply docker/vnc/Dockerfile:22]: good point

Comment (rokroskar) [reply docker/vnc/Dockerfile:22]: well I've never heard of or used `gitk` but it's an option in the git gui to visualize the history so this pretty much guarantees that a user will ask about it :)

**Review by rokroskar — APPROVED** [review]
🚀 Thanks!!

**Pull request merged**